utils/time.go

```diff
@@ -7,9 +7,10 @@ import (
 )
 
 // ToTimestamp converts time.Time to timestamp.TimeStamp.
+// Ref: https://github.com/golang/protobuf/blob/master/ptypes/timestamp.go#L113
 func ToTimestamp(time time.Time) *timestamp.Timestamp {
 	return &timestamp.Timestamp{
-		Seconds: int64(time.Second()),
+		Seconds: time.Unix(),
```

Comment:
use `ptypes.TimestampProto` and `ptypes.TimeStamp` for conversions instead. and let this function be a wrapper over that function instead